src/USER-MISC/fix_pafi.cpp

```diff
@@ -0,0 +1,798 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+/* ------------------------------------------------------------------------
+   Contributing authors: Thomas Swinburne (CNRS & CINaM, Marseille, France)
+
+   Please cite the related publication:
+   T.D. Swinburne and M.-C. Marinica, Unsupervised calculation of free energy barriers in large crystalline systems, Physical Review Letters 2018
+------------------------------------------------------------------------- */
+
+
+#include "fix_pafi.h"
+#include <mpi.h>
+#include <math.h>
```

Comment:
this is still not using C++ style include files. I am not very fussy about `mpi.h`, but for using consistently C++ style include files instead of the C style includes, there is a known benefit. in several cases there can otherwise be problem with inconsistent includes if a mix of some of these are included and some header files must include c-library headers and consequently include the C++ version.

Reply:
I have changed everything to C++ style